client/api/src/main/java/org/opennms/velocloud/client/api/model/Link.java

```diff
@@ -0,0 +1,258 @@
+            return this;
+        }
+
+        public Builder withCreated(OffsetDateTime created) {
+            this.created = created;
+            return this;
+        }
+
+        public Builder withEdgeId(final Integer edgeId) {
+            this.edgeId = edgeId;
+            return this;
+        }
+
+        public Builder withLogicalId(String logicalId) {
+            this.logicalId = logicalId;
+            return this;
+        }
+
+        public Builder withInternalId(String internalId) {
+            this.internalId = internalId;
+            return this;
+        }
+
+        public Builder withInterface(String _interface) {
+            this._interface = _interface;
+            return this;
+        }
+
+        public Builder withMacAddress(String macAddress) {
+            this.macAddress = macAddress;
+            return this;
+        }
+
+        public Builder withIpAddress(final String ipAddress) {
+            this.ipAddress = ipAddress;
+            return this;
+        }
+
+        public Builder withIpv6Address(final String ipv6Address) {
+            this.ipv6Address = ipv6Address;
+            return this;
+        }
+
+        public Builder withNetmask(final String netmask) {
+            this.netmask = netmask;
+            return this;
+        }
+
+        public Builder withNetworkSide(String networkSide) {
+            this.networkSide = networkSide;
+            return this;
+        }
+
+        public Builder withNetworkType(String networkType) {
+            this.networkType = networkType;
+            return this;
+        }
+
+        public Builder withDisplayName(String displayName) {
+            this.displayName = displayName;
+            return this;
+        }
+
+        public Builder withIsp(String isp) {
+            this.isp = isp;
+            return this;
+        }
+
+        public Builder withOrg(String org) {
+            this.org = org;
+            return this;
+        }
+
+        public Builder withLat(Double lat) {
+            this.lat = lat;
+            return this;
+        }
+
+        public Builder withLon(Double lon) {
+            this.lon = lon;
+            return this;
+        }
+
+        public Builder withLinkMode(String linkMode) {
+            this.linkMode = linkMode;
+            return this;
+        }
+
+        public Builder withAlertsEnabled(Boolean alertsEnabled) {
+            this.alertsEnabled = alertsEnabled;
+            return this;
+        }
+
+        public Builder withOperatorAlertsEnabled(Boolean operatorAlertsEnabled) {
+            this.operatorAlertsEnabled = operatorAlertsEnabled;
+            return this;
+        }
+
+        public Builder withServiceGroups(String serviceGroups) {
+            this.serviceGroups = serviceGroups;
+            return this;
+        }
+
+        public Link build() {
+            return new Link(this);
+        }
+    }
+
+    public static Builder builder() {
+        return new Builder();
+    }
+}
```

Comment:
Same here toString()

Reply:
See lines 87-110

Reply:
yes sorry I'm used to find it at the end. Didn't see it at the top :)